dotcom-rendering/src/web/components/ShowMore.importable.tsx

```diff
@@ -88,6 +84,22 @@ export const ShowMore = ({
 
 	const showMoreContainerId = `show-more-${containerId}`;
 
+	useEffect(() => {
+		/**
+		 * Focus the first of the new cards when they're loaded in.
+		 * There's no need to check `isOpen` here because if `isOpen` is
+		 * `false` then `filteredData` will be `undefined`.
+		 * */
+		if (filteredData && filteredData.length > 0) {
```

Comment:
Might be more confusing, but 0 is falsy, so you could do:

```suggestion
		if (filteredData?.length) {
```

Reply:
yeah that's a good call I think -- I'd been thinking that it was good to be more explicit, but I think we use optional chaining widely enough in the codebase now that this should be fine 👍